### PR TITLE
Update upgrade paths and env vars in tests

### PR DESCRIPTION
**Pull request by fernflower**
- 8.7->9.0 substituted with 8.8->9.0
- removed unnecessary TARGET_KERNEL env var

### CODE REVIEW

**Comment by fernflower** [comment]
Proof it works:

https://github.com/fernflower/leapp/actions/runs/4488317535

File: .github/workflows/tmt-tests.yml [hunk 1]
```diff
@@ -76,24 +76,24 @@ jobs:
...
     uses: ./.github/workflows/reuse-tests-8to9.yml
     secrets: inherit
     with:
       copr_artifacts: ${{ needs.call_workflow_copr_build.outputs.artifacts }}
       tmt_plan_regex: "^(?!.*upgrade_plugin)(?!.*tier[2-3].*)(?!.*rhsm)(?!.*c2r)(?!.*sap)(?!.*7to8)(?!.*max_sst)"
-      variables: 'SOURCE_RELEASE=8.7;TARGET_RELEASE=9.0;TARGET_KERNEL=el9;RHSM_REPOS=rhel-8-for-x86_64-appstream-rpms,rhel-8-for-x86_64-baseos-rpms;LEAPPDATA_BRANCH=upstream'
-      compose: 'RHEL-8.7.0-Nightly'
-      pull_request_status_name: "8.7to9.0"
-      tmt_context: "distro=rhel-8.7"
+      variables: 'SOURCE_RELEASE=8.8;TARGET_RELEASE=9.2;RHSM_REPOS=rhel-8-for-x86_64-appstream-rpms,rhel-8-for-x86_64-baseos-rpms;LEAPPDATA_BRANCH=upstream'
```
Comment (mm0ran) [inline .github/workflows/tmt-tests.yml:93]: This is a beta to beta upgrade so I think we need some additional changes:
```suggestion
      variables: 'SOURCE_RELEASE=8.8;TARGET_RELEASE=9.2;RHSM_REPOS=rhel-8-for-x86_64-appstream-beta-rpms,rhel-8-for-x86_64-baseos-beta-rpms;LEAPP_DEVEL_TARGET_RELEASE=9.2;LEAPPDATA_BRANCH=upstream'
```

**Review by mm0ran — DISMISSED** [review]
Other than the suggested change, LGTM!

File: .github/workflows/tmt-tests.yml (hunk 1 above)
Comment (fernflower) [reply .github/workflows/tmt-tests.yml:93]: Right, thanks. Currently it doesn't matter for upstream tests as they are non-rhsm, but let's do it right.

**Comment by fernflower** [comment]
Rerunning - https://github.com/fernflower/leapp/actions/runs/4500129902

File: .github/workflows/tmt-tests.yml (hunk 1 above)
Comment (r0x0d) [reply .github/workflows/tmt-tests.yml]: @fernflower should I update that in packit for both leapp and leapp repository too?

**Review by mm0ran — APPROVED** [review]
LGTM!

**Pull request merged**